install_redhat_gpu_drivers.sh

```diff
@@ -15,6 +15,13 @@
 # limitations under the License.
 #
 # Script should be used only as a part of Dockerfiles
+
+# Check if INSTALL_DRIVER_VERSION is set
+if [ -z "$INSTALL_DRIVER_VERSION" ]; then
```

Comment:
why not use default version? In 99% of cases the latest will be used...

Reply:
I only add validation to the script so that we never loose time debugging silent errors